src/components/input/demoErrors/index.html

```diff
@@ -27,7 +27,7 @@ <h1 class="md-toolbar-tools">
 
       <md-input-container>
         <label>Hourly Rate (USD)</label>
-        <input required type="number" step="any" name="rate" ng-model="project.rate" min="800" max="4999" required>
+        <input required type="number" step="any" name="rate" ng-model="project.rate" min="800" max="4999">
```

Comment:
Is this reflected under the "source" here https://material.angularjs.org/#/demo/material.components.input
I couldn't see that changed.